CHANGELOG.md

```diff
@@ -6,6 +6,40 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.2.0] 2019-10-21
+
+### Added
+
+- Added a new [public plugin interface](pkg/secretless/plugin) for building connector plugins
+- Added a new [public log interface](pkg/secretless/log) for standardizing logging
+- Added reporting on code coverage data to unit test output
```

Comment:
Added reporting on code coverage data to unit test output -> Added code coverage reporting to unit test output